src/Locale/LocaleManager.php

```diff
@@ -11,12 +11,12 @@
 
 namespace Flarum\Locale;
 
-use Symfony\Component\Translation\Translator as SymfonyTranslator;
+use Illuminate\Contracts\Translation\Translator;
 
 class LocaleManager
```

Comment:
LocaleManager::getTranslator docblock is still returning SymfonyTranslator.

AbstractFrontend::addTranslations depends on the SymfonyTranslator::getCatalogue, which is not present in the Illuminate contract.

I think LocaleManager needs to require a Flarum\Locale\Translator instance directly, rather than the Illuminate contract - that way we know it fulfils both the Illuminate and Symfony contracts.